package.json

```diff
@@ -86,8 +86,10 @@
     "babel-preset-react": "~6.3.13",
     "babel-preset-stage-0": "~6.3.13",
     "chai": "~3.3.0",
+    "chai-enzyme": "~0.4.1",
+    "cheerio": "~0.20.0",
```

Comment:
It's a peer dep of [chai-enzyme](https://github.com/producthunt/chai-enzyme/blob/master/package.json)